src/stemmabench/stemma_generator.py

```diff
@@ -19,6 +20,7 @@ def __init__(
         config_path: str = None,
         original_text: str = None,
         path_to_text: str = None,
+        random_state: int = None
     ) -> None:
         """A class to perform variant generation.
```

Comment:
Absolutely not your fault, but can you add the docstrings here please ?

Reply:
- [x] Complete docstrings.